### ISSUE TITLE
Document new structure

### ISSUE THREAD

**Issue by falun** [dustball/gping.io#9]
Previously there was a pretty trivial breakdown of the system -- `read.php` read from the DB and `write.php` wrote to the DB.

Things are a little more complex now and it'd be nice to capture how things work. Not looking for something super detailed as it'll remain continue to shift for a while but getting a start on how we deal with request routing and what the major DB read/write functions are would be nice.

Basically fill out `docs/www.md`.
